### PR TITLE
DCMD-327: Control the value of status

### PR DESCRIPTION
**Pull request by Xiaoming**

### CODE REVIEW

File: app/models/job.rb [hunk 1]
```diff
@@ -1,2 +1,4 @@
 class Job < ApplicationRecord
+ # validates :status, inclusion: { in: %w(success failed),
```
Comment (mikedurbin) [inline app/models/job.rb]: Unless there's a reason lets omit these commented out lines.

**Review by mikedurbin — COMMENTED** [review]
This code change does what we want.  If you would remove those unused comment line (or explain why they're helpful), commit then push them to the DCMD-327 branch, I'll merge the commit.

File: app/models/job.rb (hunk 1 above)
Comment (Xiaoming) [reply app/models/job.rb]: It seems that we should put the logic in the model. But code in the job.rb will return all the errors with the customized message and the error code won't be 400. If it's ok to leave everything in the controller, I will remove changes in job.rb.

Comment (mikedurbin) [reply app/models/job.rb]: That's a good point.  Try putting the logic in the model, then in the controller testing if it's valid then passing the validation message with the 400 response.  

There's some documentation here: http://guides.rubyonrails.org/active_record_validations.html

We definitely want to return a 400 if the resource is invalid, and a 500 if some other error (DB access or something) prevented "@job.save" from returning true.

**Pull request merged**